### ISSUE TITLE
wrong tab url loads when reopening Min

### ISSUE THREAD

**Issue by shalva97** [minbrowser/min#1279]
* **Min Version:** 1.16.0
* **Operating system:**  Arch

## Expected Behavior
lets say i went to a website and downloaded a file. then i close browser and reopen it. i expect to see the website that i downloaded the file from

## Actual Behavior
when i reopen Min last url is laoded, which is the download link and a file save dialog shows up.

## To Reproduce
download a file from a website that doesnot open new tab for downloads. then restart Min and go to that tab. you will see a file save dialog again, without option to go back in history. 

in my case it is a Russian torrent [site](http://rutor.info) :D click any release and downlaod the torrent, then reopen min and you should see file save dialog popup again

**Comment by Milo123459** [comment]
@shalva97 on the latest version, are you still experiencing the issue?

**Comment by Syndamia** [comment]
Yep, it still exists. An easy test is with (this)[https://www.thinkbroadband.com/download], what I notice is that when downloading the link in the tab bar is the one for download, although if you refresh the page it's back to the original one. I guess Min handles download links as a "semi-new tab for download" and when you restart the browser, the link that gets saved is the download one.
